src/Test/Utilities/Portable/FX/CappedStringWriter.cs

```diff
@@ -31,6 +31,7 @@ public CappedStringWriter(int expectedLength)
                 _expectedLength = expectedLength;
                 _remaining = Math.Max(256, expectedLength * 4);
             }
+            _remaining = int.MaxValue;
```

Comment:
>_remaining = int.MaxValue; [](start = 12, length = 26)

Is this additional intentional?

Reply:
No. Thanks for catching. That was a temp change to help debug a failure.